### DISCUSSION TITLE
Error while installing in Delphi

### DISCUSSION THREAD

**Discussion by phvillaverde** [helgovic/Gradle-for-Delphi#5] [General]
I'm trying to install in Delphi 11.3 and getting "required package 'AlcinoeAlexandria' not found" error. Someone could help me?

**Comment by MFERLINI** [comment]
I have the same problem with Delphi 10.4 ...

**Reply by helgovic** [reply]
You must have an old download. This has been fixed.

**Comment by MFERLINI** [comment]
Ok, I'm looking for the updated version.
Thank you very much!

Bye

Da: Helge Johnsen ***@***.***>
Inviato: venerdì 12 gennaio 2024 09:47
A: helgovic/Gradle-for-Delphi ***@***.***>
Cc: Marco Ferlini ***@***.***>; Comment ***@***.***>
Oggetto: Re: [helgovic/Gradle-for-Delphi] Error while installing in Delphi (Discussion #5)


You must have an old download. This has been fixed.
